deerlab/dipolarkernel.py

```diff
@@ -97,22 +102,22 @@ def dipolarkernel(t,r,pathinfo = 1, B = 1, method = 'fresnel', excbandwidth = in
         Lam0 = 0.5                     # unmodulated part
         lam = 0.4                      # modulation depth main signal
         lam21 = 0.1                    # modulation depth 2+1 contribution
-        tau2 = 4                       # refocusing time of 2+1 contribution
-        pathinfo = [[],[],[]]
-        pathinfo[0] = [Lam0,   nan]    # unmodulated part, gives offset
-        pathinfo[1] = [lama,    0 ]    # main modulation, refocusing at time zero
-        pathinfo[2] = [lam21, tau2]    # 2+1 modulation, refocusing at time tau2
+        tau2 = 4                       # refocusing time (us) of 2+1 contribution
+        pathinfo = []
+        pathinfo.append([Lam0])           # unmodulated part, gives offset
+        pathinfo.append([lam1,   0  ])    # main modulation, refocusing at time zero
+        pathinfo.append([lam2,  tau2])    # 2+1 modulation, refocusing at time tau2 
         
```

Comment:
`pathway = [[Lam0],[lam1,0],[lam2,tau2]]`
Also, I think `pathway = [Lam0,[lam1,0],[lam2,tau2]]` should work for usability.

Reply:
See 0f8017d